discovery-provider/src/api/v1/users.py

```diff
@@ -122,3 +128,94 @@ def get(self):
         users = response["users"]
         users = list(map(extend_user, users))
         return success_response(users)
+
+
+followers_route_parser = reqparse.RequestParser()
+followers_route_parser.add_argument('user_id', required=False)
+followers_route_parser.add_argument('limit', required=False, type=int)
+followers_route_parser.add_argument('offset', required=False, type=int)
+
+followers_response = make_response("followers_response", full_ns, fields.List(fields.Nested(user_model_full)))
+@full_ns.route("/<string:user_id>/followers")
+class FollowerUsers(Resource):
+    @record_metrics
+    @ns.expect(followers_route_parser)
+    @ns.doc(
+        id="""Get User's Followers""",
```

Comment:
I think these routes are flipped.

/id/followers - should be all the users that follow <id>
/id/following - should be all users that <id> follows

Maybe for doc text, we can say

/id/followers - All users that follow the provided user
/id/following - All users that the provided user follows

Reply:
I think it's helpful to use the dapp as a comparison. when you expand the "following" modal, you would be following all the users there